cs50/2017/fall/recover/check50/__init__.py

```diff
@@ -54,6 +55,59 @@ class Recover(Checks):
         "3510482b20a5b93eb8e8558227fa80f8460581f71e433932d11bac989e9d17fc"   # 049.jpg
     ]
 
+    sizes = [
+        45568,   # 000.jpg
+        51200,   # 001.jpg
+        20992,   # 002.jpg
+        26112,   # 003.jpg
+        62976,   # 004.jpg
+        221696,  # 005.jpg
+        183296,  # 006.jpg
+        54784,   # 007.jpg
+        45568,   # 008.jpg
+        171520,  # 009.jpg
+        35840,   # 010.jpg
+        57856,   # 011.jpg
+        173056,  # 012.jpg
+        75776,   # 013.jpg
+        49152,   # 014.jpg
+        202752,  # 015.jpg
+        22016,   # 016.jpg
+        183296,  # 017.jpg
+        31232,   # 018.jpg
+        44032,   # 019.jpg
+        194560,  # 020.jpg
+        56832,   # 021.jpg
+        220672,  # 022.jpg
+        77824,   # 023.jpg
+        25088,   # 024.jpg
+        233984,  # 025.jpg
+        235008,  # 026.jpg
+        32768,   # 027.jpg
+        44032,   # 028.jpg
+        208896,  # 029.jpg
+        39936,   # 030.jpg
+        49664,   # 031.jpg
+        278528,  # 032.jpg
+        221696,  # 033.jpg
+        47616,   # 034.jpg
+        179712,  # 035.jpg
+        20480,   # 036.jpg
+        55808,   # 037.jpg
+        202240,  # 038.jpg
+        86528,   # 039.jpg
+        56832,   # 040.jpg
+        226304,  # 041.jpg
+        210432,  # 042.jpg
+        161792,  # 043.jpg
+        21504,   # 044.jpg
+        60928,   # 045.jpg
+        16896,   # 046.jpg
+        27648,   # 047.jpg
+        162304,  # 048.jpg
+        34304    # 049.jpg
+    ]
+
```

Comment:
@tlively mind moving these to a file and loading from the file? That'll make it way easier to maintain this list as JPEGs change, since we can update a file with a (Linux) command like:

```
stat -c "%s %n" *.jpg > sizes.txt
```